### PR TITLE
build: use biomejs as linter + formatter

### PR DESCRIPTION
**Pull request by setchy**
Proposing to use biomejs.dev as an all-in-one linter and formatter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,64 +1,63 @@
...
+    "lint": "biome check --fix",
+    "test": "jest"
+  },
+  "types": "lib/index.d.ts",
+  "dependencies": {
+    "electron-positioner": "^4.1.0"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "1.8.2",
+    "@amaurym/config": "^1.3.5",
```
Comment (amaury1093) [inline package.json:47]: I'd like to remove dependency on this package (it's mine). We can probably remove it

File: tsconfig.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-	"extends": "./node_modules/@amaurym/tsconfig/tsconfig",
-	"compilerOptions": {
-		"declaration": true,
-		"module": "commonjs",
-		"outDir": "./lib",
-		"skipLibCheck": true,
-		"target": "es5"
-	}
+  "extends": "./node_modules/@amaurym/tsconfig/tsconfig",
```
Comment (amaury1093) [inline tsconfig.json:2]: actually, we should also unroll that config above, and put it explicity in this file, so remove the dependency on @amaurym/tsconfig

**Review by amaury1093 — COMMENTED** [review]
Fine to use biome, I anyways wanted to remove dependency on my own packages `@amaurym/{config,typescript,eslint}`

File: tsconfig.json (hunk 2 above)
Comment (setchy) [reply tsconfig.json]: unrolled ✅

File: package.json (hunk 1 above)
Comment (setchy) [reply package.json]: removed 🗑️

**Comment by setchy** [comment]
@amaury1093 - feedback incorporated.  

After this is merged, would you be able to release a new version?  Very interested in getting https://github.com/max-mapper/menubar/pull/479 into Gitify

**Review by amaury1093 — APPROVED** [review]
Thanks

**Pull request merged**